firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/ApkInstaller.java

```diff
@@ -85,7 +85,7 @@ void handleAppResume(Activity activity) {
 
   Task<Void> installApk(String path) {
     synchronized (installTaskLock) {
-      Activity currentActivity = lifeCycleNotifier.getCurrentActivity();
+      Activity currentActivity = lifeCycleNotifier.getForegroundActivity().getResult();
```

Comment:
Calling getResult() will throw if there's no foreground activity.

Instead should we pass the activity to this method as an argument? Then wherever this is called, we can chain it all together:

```
lifeCycleNotifier.getForegroundActivity()
  .onSuccessTask(activity -> apkInstaller.installApk(path, activity))
  ...
```

Reply:
Good catch! Fixed to implement that change.